service/middleware/access_log.go

```diff
@@ -0,0 +1,105 @@
+package middleware
+
+/* CHECKLIST
+ * [ ] Uses interfaces as appropriate
+ * [ ] Private package variables use underscore prefix
+ * [ ] All parameters validated
+ * [ ] All errors handled
+ * [ ] Reviewed for concurrency safety
+ * [ ] Code complete
+ * [ ] Full test coverage
+ */
+
+import (
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/ant0ine/go-json-rest/rest"
+
+	"github.com/tidepool-org/platform/service"
+)
+
+type AccessLog struct{}
+
+const (
+	_LogErrors           = "errors"
```

Comment:
Yes, I kept getting confused (and namespace collision errors) between private/package consts and variables versus function local variables. Adding an underscore (which forces it to be private) seemed to make it clear that it was not a function local variable. Not technically the 100% correct way in Go, but since I see value in distinguishing between function local variables and global const/vars this seemed to make sense.